.gitignore

```diff
@@ -34,6 +34,7 @@ dump.rdb
 **/less/*.css
 /staticmedia*
 .transifexrc
+python_env
```

Comment:
since the new deploy puts the `python_env` inside the code directory we need to ignore it so `git clean -ffd` doesn't wipe it on a `hotfix_deploy`